client/src/ui/organisms/placement/index.tsx

```diff
@@ -63,11 +74,17 @@ export function Placement() {
 
       gleanClick(`pong: pong->fetched ${response.status}`);
 
-      if (!response.ok) {
```

Comment:
I'm inclined to still keep this check - we know we only send the correct json if the status code is successful, so we're putting a lot of faith in the try catch to error out "correctly" if the status code is a failure